search-commons/src/main/java/no/unit/nva/search/common/ParameterValidator.java

```diff
@@ -180,7 +180,9 @@ public ParameterValidator<K, Q> fromRequestInfo(RequestInfo requestInfo) {
     var contentType = extractContentTypeFromRequestInfo(requestInfo);
     query.setMediaType(isNull(contentType) ? null : contentType.getMimeType());
     var uri = URI.create(HTTPS + requestInfo.getDomainName() + requestInfo.getPath());
-    query.setAccessRights(requestInfo.getAccessRights());
+    if (requestInfo.getHeaders().containsKey("Authorization")) {
+      query.setAccessRights(requestInfo.getAccessRights());
+    }
```

Comment:
if we ask for accessRights, without a "Authorization" entry in the header, it will throw an errror.
this code will run on authenticated and unauthenticated endpoints, so this should not be logged as an error.
FIX _test before asking_